conf/gobblin-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set Gobblin specific environment variables here.
+
+# Gobblin work dir
+export GOBBLIN_WORK_DIR=hdfs://localhost:8020/gobblin
+
+# Gobblin job configuration files
+export GOBBLIN_JOB_CONFIG_DIR=$GOBBLIN_WORK_DIR/jobs
```

Comment:
Removed. 
OK, handle the remaining issues (add multiple jars to libjars, how env. variables are handled, jt/fs arguments) in a separate ticket.